style.css

```diff
@@ -0,0 +1,32 @@
+.orange-margin {
+  border-top: 8px solid #F5C85F;
+}
+
+* {
+  font-family: 'Open Sans', sans-serif;
+}
+
+.main-images {
+  display:flex;
+  flex-direction:row;
+  justify-content:center;
+  align-items:center;
+  background-color:white;
```

Comment:
space after :

also use hex notation instead of keywords to be consistent. in this case `#FFF`